app/components/blacklight/response/pagination_component.rb

```diff
@@ -8,10 +8,14 @@ class PaginationComponent < Blacklight::Component
       # @param [Hash] html html options for the pagination container
       def initialize(response:, html: {}, **pagination_args)
         @response = response
```

Comment:
ViewComponents [no longer allows](https://github.com/ViewComponent/view_component/pull/1666) `t()` to be called in an initializer.